Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:22.04 as build
+
+LABEL org.opencontainers.image.authors="Santiago <san@san.com>"
```

Comment:
do we need this?

Reply:
No, we don't need it, just habit for standards.